code/enumerable/inject-sum-vs-block.rb

```diff
@@ -0,0 +1,19 @@
+require "benchmark/ips"
+
+User  = Struct.new(:name)
+ARRAY = (1..10_000_000).to_a
+
+def fast
+  ARRAY.inject(&:+)
+end
+
+def slow
+  ARRAY.inject(0) { |a, i| a + i }
```

Comment:
This should be `ARRAY.inject { |a, i| a + i }`, or the other should be `ARRAY.inject(0, :+)`